sbin/pfmon

```diff
@@ -382,6 +384,24 @@ sub normal_sighandler {
     $running = 0;
 }
 
+=head2 reload_config
+
+=cut
+
+sub reload_config {
+    if ( pf::cluster::is_management ) {
+        $process = $TRUE;
```

Comment:
No need to signal the childs, we only tell the parent to run (or not to run) the childs